src/autorest-core/test/end-to-end.ts

```diff
@@ -82,7 +82,7 @@ import { PumpMessagesToConsole } from './test-utility';
 
     autoRest.Message.Subscribe((_, m) => { if (m.Channel === Channel.Warning || Channel.Error) { messages.push(m); } });
     assert.equal(await autoRest.Process().finish, true);
-    assert.equal(messages.length, 0);
+    assert.notEqual(messages.length, 0);
```

Comment:
if this is the fix you talk about [here](https://github.com/Azure/autorest/pull/2334/files/2992d5a101a708f6272d92ecc990c3da3b03263d#diff-f1316e08d886e44821dda7ed03690a01R73), that comment may wanna be updated too ;)